Automattic/wp-calypso
# Revert "Receipts: Convert receipt assembler to TypeScript"

#68857 is causing the post-checkout redirect to fail for Jetpack products. This reverts it until we can figure out what's happening.

Reverts Automattic/wp-calypso#68857

See: p1665663818263299-slack-C0117V2PCAE

**To test:**
- sandbox store
- spin up a jurassic.ninja site and attach it to your account
- in Calypso, click "Upgrades" and purchase a product (e.g. Jetpack Backup)
- complete checkout and verify that you are successfully redirected after Checkout (most likely to wp-admin)
- on a WPcom site, visit "Upgrades > Plans" and purchase a plan
- complete checkout and verify that you are successfully redirected after Checkout (most likely to a Calypso Thank You page)
